.nix/devnet-gce.nix

```diff
@@ -16,82 +15,78 @@ in {
       };
     };
   };
-  "${machine-name}" = { pkgs, resources, ... }: {
-    deployment = {
-      targetEnv = "gce";
-      gce = gce-input // {
-        machineName = machine-name;
-        network = resources.gceNetworks.composable-devnet;
-        region = "europe-central2-c";
-        instanceType = "n2-standard-4";
-        rootDiskSize = 200;
-        tags = [ "http" "https" ];
+  "${machine-name}" = args@{ pkgs, resources, ... }:
+    ({
+      deployment = {
+        targetEnv = "gce";
+        gce = (gce-input // {
+          inherit region;
+          machineName = machine-name;
+          network = resources.gceNetworks.composable-devnet;
+          instanceType = "n2-standard-4";
```

Comment:
if we consider having ibc and xcvm stuff soon here to play with, `n2-standard-8` is safer not to worry in future. also it is safer for demos and people plaiting around. if this is machine all in one networks and ui and indexers.

Reply:
This should not be maintained by the dev team and propely moved off to SRE/DevOps to deploy machines etc via a more stable tool like terraform